aries-core/src/js/components/core/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Box } from 'grommet';
+
+export const Sidebar = ({ children, header, ...rest }) => {
+  return (
+    <Box
+      pad="medium"
+      gap="large"
+      fill="vertical"
+      width="auto"
+      height="100%"
+      {...rest}
+    >
+      {header}
+      {children}
```

Comment:
Since all of the Navigation examples have the same wrapper, I'm wondering if Sidebar should handle that aspect? I'm thinking that Sidebar could do this instead of just `{children}`: `<Box justify="between"><Box>{children[0 - next to last]}</Box>{children[last]}</Box>`. That would simplify the children in the examples to just `Nav` components containing `Button` and `Anchor`.

Reply:
yea, we can let Sidebar do more work on its side.
Can we just use `<Box justify="between">{children}</Box>`? 
wrapping the children with additional internal Box may be redundant unless we want to wrap with a Nav component instead, and wrapping with a Nav felt like a too wide assumption (I had it like that at the beginning)

Note: I just verified that the Fragment renders well on the Sidebar `justify="between"` styling.